src/odinapi/run.py

```diff
@@ -0,0 +1,6 @@
+from .blueprints import register_blueprints
+from .api import app
+
+register_blueprints(app)
+if __name__ == "__main__":
+    app.run()
```

Comment:
```suggestion
    app.run()

```

Reply:
Cannot se what change you are requesting. The idea here is to make it possible to start flask with run from the command line, but when gunicorn starts, it just uses the app object.